packages/ui/sass/components/inputs.scss

```diff
@@ -7,10 +7,6 @@ $input-border: 2px solid #ccc;
 
   &--inline {
     flex-direction: row;
-
-    @include breakpoint(phone-only) {
```

Comment:
Why did we delete this breakpoint?

Reply:
Wasn't requried it was breaking the UI in mobile we already have flex-direction: column in parent so we don't need this.